docs/content_management/create_edit_pages.md

```diff
@@ -157,12 +157,26 @@ The following actions are available:
 - Move up block - allows you to change position of the block on the page by moving it up
 - Move down block - allows you to change position of the block on the page by moving it down
 - Block settings - allows you to access properties window
+- Copy - allows you to reuse a configured block across pages by using the Clipboard section.
 - Duplicate - duplicates existing block with its settings
 - Refresh - refreshes preview of the block
 - Delete - deletes existing block
 
 ![Block window - size adjustment](img/block_window_size_adjust.png "Block window - size adjustment")
 
+#### Copy blocks
+
+You can copy blocks to reuse them on other pages.
+
+When you copy a block, it gets saved in the "Clipboard" section of the Page Blocks toolbox.
+From there you can drag and drop the block to the page like any other block.
+Doing so puts an exact copy of the block on the page, with all its settings, as it was when you copied it.
+
+The clipboard can store only one block at a time.
+The content of the clipboard remains accessible while editing different pages and is visible only to you.
```

Comment:
```suggestion
Contents of the clipboard are visible only to you, and...
```
"preserve switching into a new session"?
"remain accessible until the end of session"?

Reply:
I've chosen `preserve switching into a new session`